pandas/tests/frame/test_stack_unstack.py

```diff
@@ -1175,6 +1175,32 @@ def test_stack_timezone_aware_values():
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize("dropna", [True, False])
+def test_stack_empty_frame(dropna):
+    # GH 36113
+    expected = Series(index=MultiIndex([[], []], [[], []]), dtype=np.float64)
+    result = DataFrame(dtype=np.float64).stack(dropna=dropna)
+    tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("dropna", [True, False])
+@pytest.mark.parametrize("fill_value", [None, 0])
+def test_stack_unstack_empty_frame(dropna, fill_value):
+    # GH 36113
+    result = (
+        DataFrame(dtype=np.intp).stack(dropna=dropna).unstack(fill_value=fill_value)
+    )
+    expected = DataFrame(dtype=np.intp)
```

Comment:
these might be expected int64 (and same below), check the 32-bit build. The reason is we round trip to Int64 as an index.